pages/en/developer/graphql-api.mdx

```diff
@@ -145,11 +161,23 @@ _lte
 _in
 _not_in
 _contains
+_contains_nocase
 _not_contains
+_not_contains_nocase
 _starts_with
+_starts_with_nocase
 _ends_with
+_ends_with_nocase
 _not_starts_with
+_not_starts_with_nocase
 _not_ends_with
+_not_ends_with_nocase
+```
+
+In addition, the following global filters are available as part of `where` argument:
+
+```gr
+_change_block
```

Comment:
is it worth mentioning that there is only one supported filter here (`number_gte`)?

Reply:
sure, done